docs/docs/00.md

```diff
@@ -52,4 +52,8 @@ usersList:
     name: airhacks.com
     img: https://airhacks.com/logo.svg
     url: https://airhacks.com
+  - tongdun:
+    name: tongdun
+    img: https://www.tongdun.cn/static/favicon.ico
```

Comment:
do you have a higher-resolution logo? It looks rather blurry

<img width="730" alt="image" src="https://user-images.githubusercontent.com/39992/91504542-49d04580-e882-11ea-8116-24ce9a0aa872.png">